src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,51 @@
+.clock {
+  position: relative;
+  z-index: -3;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  @include clock-size($clock-side);
+
+  border-radius: 50%;
+  border: 11px dotted brown;
+  background-color: antiquewhite;
+
+  &__center {
+    @include center-size($center-side);
+
+    border-radius: 50%;
+    background-color: brown;
+  }
+
+  &__minute-arrow {
+    position: absolute;
```

Comment:
you could add a mutual class for  minute and arrow and not write the same properties twice (position and transform-origin)

Reply:
Done!